jupyterlab_gcpscheduler/src/data.ts

```diff
@@ -99,19 +103,46 @@ export const CONTAINER_IMAGES: Option[] = [
     value: `${GCR_PREFIX}/tf2-gpu.2-1:latest`,
     text: 'TensorFlow 2.1 (GPU)',
   },
+  {
+    value: `${GCR_PREFIX}/tf2-cpu.2-3:latest`,
+    text: 'TensorFlow 2.3 (CPU only)',
+  },
+  {
+    value: `${GCR_PREFIX}/tf2-gpu.2-3:latest`,
+    text: 'TensorFlow 2.3 (GPU)',
+  },
   {
     value: `${GCR_PREFIX}/pytorch-cpu.1-4:latest`,
     text: 'PyTorch 1.4 (CPU only)',
   },
   {
-    value: `${GCR_PREFIX}/pytorch-gpu.1-4:latest`,
-    text: 'PyTorch 1.4 (GPU)',
+    value: `${GCR_PREFIX}/pytorch-gpu.1-6:latest`,
+    text: 'PyTorch 1.6 (GPU)',
+  },
+  {
+    value: `${GCR_PREFIX}/xgboost-cpu:latest`,
+    text: 'RAPIDS XGBoost',
   },
   {
     value: `${GCR_PREFIX}/r-cpu.3-6:latest`,
     text: 'R 3.6 (with r-essentials)',
   },
+  {
+    value: `${GCR_PREFIX}/base-cpu:latest`,
+    text: 'Python 2 and 3',
+  },
   { value: `${GCR_PREFIX}/beam-notebooks:latest`, text: 'Apache Beam' },
+  {
+    value: 'gcr.io/cloud-dataproc/dataproc-spawner:prod',
```

Comment:
I don't think we should actually be supporting the Dataproc Spawner image. It's not actually a Notebook image but an image used to spawn a Dataproc Cluster which then hosts JupyterLab.

https://cloud.google.com/dataproc/docs/tutorials/dataproc-hub-users

Reply:
Removed